src/petals/client/remote_forward_backward.py

```diff
@@ -88,8 +88,8 @@ async def run_remote_forward(
     # Modify forward_schema to support prompts
     args_schema, kwargs_schema = rpc_info["forward_schema"]
     # TODO: rm this assert when support arbitrary number of input tensors
-    assert len(args_schema) == 1 and len(inputs) == 2
-    forward_schema_with_prompts = (tuple(args_schema * len(inputs)), kwargs_schema)
+    assert len(args_schema) == 2 and len(inputs) == 3
```

Comment:
from conversation of @artek0chumak vs @borzunov vs @justheuristic 

We need to support - temporarily - an edge case where old version client interacts with new version server and vice versa.
Since the code is temporary, it doesn't have to be perfectly elegant.

If you want, I can put together a PR into your PR to covert these two cases.

__Case 1:__ old client vs new server
- we can check for the number of inputs in each `rpc_*` method, and, if not given, create a dummy tensor right there and then
- in two weeks' time, we can remove this check

__Case 2:__ new client vs old server
- server expects exactly two arguments and does not have a futureproofing mechanism
- possible solution: simply do *not* send attention mask unless it is explicitly provided
   - in 2 weeks time, switch to sending mask all the time
- in theory, we could change the protocol so that client would check the server version before request. I would argue that it is an overkill. Then again, it's @artek0chumak 's choice.